src/Symfony/Component/Messenger/Bridge/Redis/Tests/Transport/ConnectionTest.php

```diff
@@ -398,4 +398,36 @@ public function testLastErrorGetsCleared()
 
         $this->assertSame('xack error', $e->getMessage());
     }
+
+    public function testSentinelEnabledWithRedisInstance()
+    {
+        $this->expectException(\InvalidArgumentException::class);
+        $this->expectExceptionMessage('Cannot configure Redis Sentinel and Redis instance at the same time');
+
+        $redis = $this->createMock(\Redis::class);
+        Connection::fromDsn('redis://localhost/messenger-clearlasterror', ['delete_after_ack' => true, 'sentinel_master' => 'any'], $redis);
+    }
+
+    public function testSentinelEnabledWithRedisClusterInstance()
+    {
+        $this->expectException(\InvalidArgumentException::class);
+        $this->expectExceptionMessage('Cannot configure Redis Sentinel and Redis Cluster instance at the same time');
+
+        $redis = $this->createMock(\RedisCluster::class);
+        Connection::fromDsn('redis://localhost/messenger-clearlasterror', ['delete_after_ack' => true, 'sentinel_master' => 'any'], $redis);
+    }
+
+    public function testInvalidSentinelMasterName()
+    {
+        $master = getenv('MESSENGER_REDIS_SENTINEL_MASTER');
+        if (!(bool) $master) {
```

Comment:
this should be `if (!getenv('MESSENGER_REDIS_SENTINEL_MASTER')) {`
I suppose it's expected that the env var is not used in this test case?